backend/TraderX/src/main/java/cmpe451/group6/rest/template/model/TemplateItem.java

```diff
@@ -0,0 +1,77 @@
+package cmpe451.group6.rest.template.model;
+
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.List;
+
+/**
+ *  Here, you create an object which is
+ *  going to be stored in database directly.
+ *
+ *  [DO NOT CHANGE OR IMPLEMENT THOSE CLASSES / INTERFACES. FOR TEMPLATING PURPOSE ONLY.]
+ */
+@Entity(name = "OptionalTableName")
+public class TemplateItem {
+
```

Comment:
adding @Data label to the beginning of that class is automatically creates getter and setter methods AFAIK

Reply:
IMO, creating getters & setters is not a big deal thanks to the IDE shortcuts. 
Hence that's not that necessary when we already have dozens of annotations like SwaggerAPI etc.